gk/main.c

```diff
@@ -89,14 +89,20 @@ priority_from_delta_time(uint64_t present, uint64_t past)
 }
 
 static struct gk_fib *
-look_up_fib(struct gk_lpm *ltbl, struct ip_flow *flow)
+__look_up_fib(struct gk_lpm *ltbl, struct ip_flow *flow, const char *calledby)
 {
 	int fib_id;
 
 	if (flow->proto == RTE_ETHER_TYPE_IPV4) {
 		fib_id = lpm_lookup_ipv4(ltbl->lpm, flow->f.v4.dst.s_addr);
-		if (fib_id < 0)
+		if (fib_id < 0) {
+			if (fib_id == -ENOENT) {
+				GK_LOG(WARNING,
+					"%s could not find FIB entry when processing incoming IPv4 flow\n",
+					calledby);
+			}
```

Comment:
This log entry is in a very hot path. When there's a flood attack with spoofed addresses, this log entry will come so often that it will simply disable log for the GK blocks. So it's better to promote the log level to `DEBUG`.